From 2d3f1bf6ffe56f664b498ef0ad12bb6d74cad951 Mon Sep 17 00:00:00 2001
From: Xiubo Li <Li.Xiubo@freescale.com>
Date: Wed, 30 Apr 2014 18:10:05 +0800
Subject: [PATCH] ASoC: fsl: Allow to select SAI device individually

This will be useful for out-of-tree drivers since in-tree drivers
could select it automatically.

Signed-off-by: Xiubo Li <Li.Xiubo@freescale.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit b71fc4e6c9eb1b5ce1c0861825835da273367104)
---
 sound/soc/fsl/Kconfig | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 1e120ac..6632059 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -3,9 +3,14 @@ menu "SoC Audio for Freescale CPUs"
 comment "Common SoC Audio options for Freescale CPUs:"
 
 config SND_SOC_FSL_SAI
-	tristate
+	tristate "Synchronous Audio Interface (SAI) module support"
 	select REGMAP_MMIO
 	select SND_SOC_GENERIC_DMAENGINE_PCM
+	help
+	  Say Y if you want to add Synchronous Audio Interface (SAI)
+	  support for the Freescale CPUs.
+	  This option is only useful for out-of-tree drivers since
+	  in-tree drivers select it automatically.
 
 config SND_SOC_FSL_SSI
 	tristate "Synchronous Serial Interface module support"
-- 
1.8.0

